wpimath/src/main/native/include/frc/controller/LinearPlantInversionFeedforward.h

```diff
@@ -59,6 +59,8 @@ class LinearPlantInversionFeedforward {
     Reset(m_r);
   }
 
+  LinearPlantInversionFeedforward(const LinearPlantInversionFeedforward&) =
```

Comment:
If you added a copy constructor, you also need to add the copy assignment operator.